api/src/main/java/com/codeforcommunity/dto/school/SchoolIdListResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.codeforcommunity.dto.school;
+
+import java.util.List;
+import java.util.Set;
+
+public class SchoolIdListResponse {
+  private Set<Integer> ids;
```

Comment:
Can we make this 'schoolIds` instead of `ids` just to make it clear in the response what we're looking at